zerver/openapi/zulip.yaml

```
@@ -14791,10 +14802,19 @@ paths:
                           password combination.
 
                           [ldap-auth]: https://zulip.readthedocs.io/en/latest/production/authentication-methods.html#ldap-including-active-directory
-                      realm_uri:
+                      realm_url:
                         type: string
                         description: |
                           The organization's canonical URL.
+                          **Changes**: New in Zulip 8.0 (feature level 192). The name `realm_url`
```

Needs a blank line so that the "Changes:" heading is the start of a new paragraph:
```suggestion
                          The organization's canonical URL.

                          **Changes**: New in Zulip 8.0 (feature level 192). The name `realm_url`
```